```
[ERROR] hudson/cli/DeleteBuildsCommand:58 hudson/cli/DeleteBuildsCommand must not be used
[ERROR] hudson/cli/DeleteBuildsCommand:69 hudson/cli/DeleteBuildsCommand must not be used
[ERROR] hudson/cli/HelpCommand:56 hudson/cli/client/Messages must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$1:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:38 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:39 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:38 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:38 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:38 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand$Format:38 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:42 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:42 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:49 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:51 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:71 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:84 hudson/cli/ListChangesCommand must not be used
[ERROR] hudson/cli/ListChangesCommand:86 hudson/cli/ListChangesCommand must not be used
[ERROR] jenkins/model/NewViewLink$1:25 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink$1:25 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink$1:29 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink$1:29 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink$1:38 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink$1:38 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink:25 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/model/NewViewLink:25 jenkins/model/NewViewLink must not be used
[ERROR] jenkins/security/CustomClassFilter$Static:95 jenkins/security/CustomClassFilter$Static must not be used
[ERROR] jenkins/security/CustomClassFilter$Static:102 jenkins/security/CustomClassFilter$Static must not be used
[ERROR] jenkins/security/CustomClassFilter$Static:104 jenkins/security/CustomClassFilter$Static must not be used
[ERROR] jenkins/security/CustomClassFilter$Static:107 jenkins/security/CustomClassFilter$Static must not be used
[ERROR] jenkins/security/CustomClassFilter$Static:113 jenkins/security/CustomClassFilter$Static must not be used
[ERROR] jenkins/security/CustomClassFilter$Static:118 jenkins/security/CustomClassFilter$Static must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter$1:54 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter$1:56 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter$1:57 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter:54 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter:54 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter:63 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter:43 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter:48 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/security/s2m/DefaultFilePathFilter:50 jenkins/security/s2m/DefaultFilePathFilter must not be used
[ERROR] jenkins/tasks/SimpleBuildStep$LastBuildActionFactory:105 jenkins/tasks/SimpleBuildStep$LastBuildActionFactory must not be used
```

LOL, will try to fix.

--- a/cli/src/main/java/hudson/cli/CLI.java
+++ b/cli/src/main/java/hudson/cli/CLI.java
@@ -809,9 +809,14 @@ public class CLI implements AutoCloseable {
         return authenticate(Collections.singleton(key));
     }
 
+    /** For access from {@code HelpCommand}. */
+    static String usage() {
+        return Messages.CLI_Usage();
+    }
+
     private static void printUsage(String msg) {
         if(msg!=null)   System.out.println(msg);
-        System.err.println(Messages.CLI_Usage());
+        System.err.println(usage());
     }
 
     static final Logger LOGGER = Logger.getLogger(CLI.class.getName());

--- a/core/src/main/java/hudson/cli/DeleteBuildsCommand.java
+++ b/core/src/main/java/hudson/cli/DeleteBuildsCommand.java
@@ -31,14 +31,14 @@ import java.io.PrintStream;
 import java.util.HashSet;
 import java.util.List;
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 /**
  * Deletes builds records in a bulk.
  *
  * @author Kohsuke Kawaguchi
  */
-@Restricted(DoNotUse.class) // command implementation only
+@Restricted(NoExternalUse.class) // command implementation only
 @Extension
 public class DeleteBuildsCommand extends RunRangeCommand {
     @Override

--- a/core/src/main/java/hudson/cli/HelpCommand.java
+++ b/core/src/main/java/hudson/cli/HelpCommand.java
@@ -53,7 +53,7 @@ public class HelpCommand extends CLICommand {
     protected int run() throws Exception {
         if (!Jenkins.getActiveInstance().hasPermission(Jenkins.READ)) {
             throw new AccessDeniedException("You must authenticate to access this Jenkins.\n"
-                    + hudson.cli.client.Messages.CLI_Usage());
+                    + CLI.usage());
         }
 
         if (command != null)

--- a/core/src/main/java/hudson/cli/ListChangesCommand.java
+++ b/core/src/main/java/hudson/cli/ListChangesCommand.java
@@ -15,14 +15,14 @@ import java.io.IOException;
 import java.io.PrintWriter;
 import java.util.List;
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 /**
  * Retrieves a change list for the specified builds.
  *
  * @author Kohsuke Kawaguchi
  */
-@Restricted(DoNotUse.class) // command implementation only
+@Restricted(NoExternalUse.class) // command implementation only
 @Extension
 public class ListChangesCommand extends RunRangeCommand {
     @Override

--- a/core/src/main/java/jenkins/model/NewViewLink.java
+++ b/core/src/main/java/jenkins/model/NewViewLink.java
@@ -9,10 +9,10 @@ import java.util.Collections;
 import java.util.List;
 
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 @Extension
-@Restricted(DoNotUse.class)
+@Restricted(NoExternalUse.class)
 public class NewViewLink extends TransientViewActionFactory {
 
     @VisibleForTesting

--- a/core/src/main/java/jenkins/security/CustomClassFilter.java
+++ b/core/src/main/java/jenkins/security/CustomClassFilter.java
@@ -44,7 +44,6 @@ import jenkins.model.Jenkins;
 import jenkins.util.SystemProperties;
 import org.apache.commons.io.IOUtils;
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.DoNotUse;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 /**
@@ -83,7 +82,7 @@ public interface CustomClassFilter extends ExtensionPoint {
      * Entries may also be preceded by {@code !} to blacklist.
      * Example: {@code -Dhudson.remoting.ClassFilter=com.google.common.collect.LinkedListMultimap,!com.acme.illadvised.YoloReflectionFactory$Handle}
      */
-    @Restricted(DoNotUse.class)
+    @Restricted(NoExternalUse.class)
     @Extension
     public class Static implements CustomClassFilter {
 

--- a/core/src/main/java/jenkins/security/s2m/DefaultFilePathFilter.java
+++ b/core/src/main/java/jenkins/security/s2m/DefaultFilePathFilter.java
@@ -30,7 +30,7 @@ import hudson.remoting.ChannelBuilder;
 import jenkins.ReflectiveFilePathFilter;
 import jenkins.security.ChannelConfigurator;
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 import java.io.File;
 import java.util.logging.Level;
@@ -39,7 +39,7 @@ import java.util.logging.Logger;
 /**
  * Blocks agents from writing to files on the master by default (and also provide the kill switch.)
  */
-@Restricted(DoNotUse.class) // impl
+@Restricted(NoExternalUse.class) // impl
 @Extension public class DefaultFilePathFilter extends ChannelConfigurator {
 
     /**

--- a/core/src/main/java/jenkins/tasks/SimpleBuildStep.java
+++ b/core/src/main/java/jenkins/tasks/SimpleBuildStep.java
@@ -52,7 +52,7 @@ import jenkins.model.DependencyDeclarer;
 import jenkins.model.RunAction2;
 import jenkins.model.TransientActionFactory;
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 /**
  * A build step (like a {@link Builder} or {@link Publisher}) which may be called at an arbitrary time during a build (or multiple times), run, and be done.
@@ -103,7 +103,7 @@ public interface SimpleBuildStep extends BuildStep {
     }
 
     @SuppressWarnings("rawtypes")
-    @Restricted(DoNotUse.class)
+    @Restricted(NoExternalUse.class)
     @Extension
     public static final class LastBuildActionFactory extends TransientActionFactory<Job> {
 

